### ISSUE TITLE
Add a guide for mapping a scripty script to a VS command

### ISSUE THREAD

**Issue by StingyJack** [StingyJack/Scripty#6]
For (dirty) macro or utility purposes

Perhaps what is needed for https://github.com/daveaglick/Scripty/issues/70  - effectively running scripty as a custom tool without being bound to a project.

**Comment by dstrohschein** [comment]
Exactly! I am working on a code fix for this now, as I think being able to opt for something within the solution is more usable than being bound to the project. In my use case: I have a Project in the solution that is dedicated Roslyn Scripts, executed by Scripty, to manipulate the files in the other projects based on database changes that occur (to cut down on boiler plate code writing). I don't like moving files from project to another when I know Scripty could just put them where they are intended at run-time.

**Comment by StingyJack** [comment]
I dont think we even need to be bound to a solution being loaded.  A script can check if there is one and use it, but I haven't seen where a script _needs_ a solution in order to run.

**Comment by dstrohschein** [comment]
I agree with that - but the code I sent in as a PR assumes there is one - I took the easiest route. It could be expanded to do what you are suggesting - I just took advantage of the fact that there is already a projectItem declared in the ScriptEngine.cs main function and went on from there. 

I might look at adding your suggestion later on this week - it makes sense, and could help me with another facet of my larger project...
